src/main/java/org/folio/rest/impl/SamlAPI.java

```diff
@@ -663,4 +660,12 @@ private boolean isInvalidOrigin(String origin) {
     return origin == null || origin.isBlank() || origin.trim().contentEquals("*");
   }
 
+  static String getCqlUserQuery(String userPropertyName, String value) {
+    // very sad that RMB does not have an option to reject fields with no index
+    List<String> supported = List.of("barcode", "externalsystemid", "id", "username", "personal.email");
+    if (!supported.contains(userPropertyName.toLowerCase())) {
```

Comment:
Field names are case sensitive, users.jsonb->>'externalsystemid' is different from users.jsonb->>'externalSystemId'

Reply:
That comes as a surprise to as CQL field names should be case insensitive.  http://docs.oasis-open.org/search-ws/searchRetrieve/v1.0/os/part5-cql/searchRetrieve-v1.0-os-part5-cql.html#_Toc324166800

But I'll be happy to make the check more strict.